src/docs/content/observability/monitoring.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Monitoring"
+title: "Grafana Monitoring"
```

Comment:
Will be better to keep just monitoring, as Grafana is just the presenter. And you have other components like prometheus, metrics-server and other"monitoring" endpoints.

Reply:
If you feel that will have other topics under monitoring and "Monitoring with Grafana is a sub-topic, propose move away from Extras and create a main topic for Monitoring and include the sub-topics there.

Reply:
I have moved Grafana monitoring under Observability (It was under extras before). I named it as grafana monitoring because we are introducing OCI monitoring as well, just to differentiate between the two. Please suggest a better name. open source monitoring or grafana-prometheus-monitoring ?
cc: @mvandervliet

Reply:
With that terms, `Grafana Monitoring` sounds fine. For grafana we do not use only prometheus as source anyways.